api-client/src/utils/geoJson.ts

```diff
@@ -8,17 +8,26 @@ export const embedInCollection = (features: PoiFeature[]): PoiFeatureCollection
   features,
 })
 
-export const prepareFeatureLocation = (poi: PoiModel, userLocation: LocationType | null): PoiFeature | null => {
+export const prepareFeatureLocation = (poi: PoiModel, userLocation: LocationType | null): PoiFeature => {
   const { featureLocation } = poi
-  if (userLocation && featureLocation) {
-    const distanceValue = distance(userLocation, featureLocation.geometry.coordinates).toFixed(1)
+  if (userLocation) {
+    const distanceValue = distance(userLocation, featureLocation.geometry.coordinates).toFixed(0)
```

Comment:
Why did you choose to change this? I think we should perhaps differ between small values < 1 or 2 (one digit after comma) and bigger ones (no digit after comma).

Reply:
oh. wasn't intentional. will revert.